projects/centre_console/rules.mk

```diff
@@ -9,7 +9,9 @@
 $(T)_DEPS := ms-common ms-helper ms-drivers
 
 $(T)_test_main_event_generator_MOCKS := get_pedal_state \
-																				drive_fsm_get_global_state \
-																				power_fsm_get_current_state \
-																				get_global_speed_state \
-																				get_global_charging_state
+										drive_fsm_get_global_state \
+										power_fsm_get_current_state \
+										get_global_speed_state \
+										get_global_charging_state
```

Comment:
The indentation messiness seems to be because Arshan used tabs, could you replace them with spaces? (Unless doing so breaks Make, which is entirely possible, in which case never mind.)